index-graphclust.md

```diff
@@ -30,32 +30,35 @@ Interactive Tours are available for Galaxy and Galaxy-GraphClust. To run the tou
 
-GraphClust pipeline for clustering similar RNA sequences together is a complex pipeline, for details please check GraphClust publication. Overall it consists of three major phases: a) sequence based pre-clustering b) encoding predicted RNA structures as graph features c) iterative fast candidate clustering then refinement
+GraphClust pipeline for clustering similar RNA sequences together is a multi-step rather complex pipeline. Overall it consists of three major phases: a) sequence based pre-clustering b) encoding predicted RNA structures as graph features c) iterative fast candidate clustering then refinement
 
-![GraphClust pipeline overview (Heyne et al. 2012)](/assets/media/graphclust_pipeline.png)
+<!-- ![GraphClust pipeline overview (Heyne et al. 2012)](./assets/img/graphclust_pipeline.png)
 
 *GraphClust pipeline overview (Heyne et al. 2012)*
+ -->
+![GraphClust-2 workflow overview](https://github.com/BackofenLab/GraphClust-2/blob/master/assets/img/figure-pipeline_zigzag.png) 
+
+Below is a coarse-grained correspondence list of GraphClust2 tool names with each step:
 
-Below is the correspondence list of Galaxy-GraphClust tool names with each step of GraphClust:
 
 |   Stage  | Galaxy Tool Name | Description|
 | :--------------------: | :--------------- | :----------------|
 |1 | [Preprocessing](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_preprocessing/preproc/0.5){:target="_blank"} | Input preprocessing (fragmentation)|
 |2 | [fasta_to_gspan](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_fasta_to_gspan/gspan/0.4){:target="_blank"} | Generation of structures via RNAshapes and conversion into graphs|
 |3 | [NSPDK_sparseVect](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/nspdk_sparse/9.2.3){:target="_blank"} | Generation of graph features via NSPDK |
 |4| [NSPDK_candidateClusters](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/NSPDK_candidateClust/9.2.3){:target="_blank"} | min-hash based clustering of all feature vectors, output top dense candidate clusters|
-|5| premLocarana,locarana_best_subtree, CMfinder | Locarna based clustering of each candidate cluster, all-vs-all pairwise alignments, create multiple alignments along guide tree, select best subtree,|
+|5| [PGMA_locarna](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_prepocessing_for_mlocarna/preMloc/0.4){:target="_blank"},[locarna](https://usegalaxy.eu/tool_runner?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_mlocarna/locarna_best_subtree/0.4){:target="_blank"}, [CMfinder](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_cmfinder/cmFinder/0.4){:target="_blank"} | Locarna based clustering of each candidate cluster, all-vs-all pairwise alignments, create multiple alignments along guide tree, select best subtree, and refine alignment.|
```

Comment:
Should these all be graphclust links? And you probably want to add `{:target="_top"}` to the end of the link not `_blank`

Reply:
True. Thanks